django_messages/models.py

```diff
@@ -51,7 +51,7 @@ class Message(models.Model):
     """
     subject = models.CharField(_("Subject"), max_length=140)
     body = models.TextField(_("Body"))
-    sender = models.ForeignKey(AUTH_USER_MODEL, related_name='sent_messages', verbose_name=_("Sender"), on_delete=models.SET_NULL)
+    sender = models.ForeignKey(AUTH_USER_MODEL, related_name='sent_messages', verbose_name=_("Sender"), on_delete=models.PROTECT)
```

Comment:
I think I like the PROTECT rule better than SET_NULL!
Can we add a migration?

Reply:
Yes, I forgot that the sender can't be null! Sorry!!